product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "Preparing your Azure account"
+
+---
+
+BigAnimal requires you to configure your Azure subscription, before you deploy your clusters. The configurations that you perform ensure that your Azure subscription is prepared to meet your clusters' resource requirements, such as: 
+
+ * Are the necessary Azure Service Providers registered for your subscription?
+ * Is there a restriction on SKUs for the Standard Esv3 and Standard Dv2 VM size families?
+ * Is there a sufficient limit on the number vCPU or Public IP addresses in your region?
+  
+!!! Note
+    Before proceeding, see [Understanding requirements in Azure](../01_preparing_azure/01_understanding_qotas_in_azure) for information about planning for your clusters' resource requirements in Azure.
+
+## Step 1: Check for requirements
+
+EDB recommends using the `azure-preflight` script to check whether all resource requirements are met in your subscription. However, you can also manually check the requirements using the Azure CLI or the Azure Portal.
```

Comment:
In general, we have rename this script from 

- `azure-preflight.sh` --> `biganimal-preflight-azure`
- the link has become https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/biganimal-preflight-azure 

so we need to substitute(find and replace) all the names and links in this PR.

Reply:
Replaced.